petab/problem.py

```diff
@@ -1,13 +1,15 @@
 """PEtab Problem class"""
 
 import os
+import tempfile
 from warnings import warn
 
 import pandas as pd
+import libcombine
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'libcombine'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4966219)